quickstarts/loadmill/config.yml

```
@@ -0,0 +1,43 @@
+# Sets the URL name of the quickstart on public I/O (required)
+slug: loadmill
+
+# Displayed in the UI (required)
+title: Loadmill Tests Results
+
+# Long-form description of the quickstart (required)
+description: |
+  Monitor your Loadmill's test results.
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  Monitor your Loadmill's test results
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Authors of the quickstart (required)
+authors:
+  - Loadmill L.T.D
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - tests
+  - loadmill
+  - API tests
+  - automation
+  - AI
+  - network
+
+# Reference to dashboards to be included in this quickstart
+dashboards:
+  - loadmill
+
+# Documentation references
+documentation:
+  - name: Loadmill Docs
+    url: docs.loadmill.com
```

Our validation expects the URl to be include the protocol (`https://`):
```suggestion
    url: https://docs.loadmill.com
```